code/_helpers/unsorted.dm

```diff
@@ -1117,3 +1119,14 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 
 /proc/pass()
 	return
+
+//clicking to move pulled objects onto assignee's turf/loc
+/proc/do_pull_click(mob/user, atom/A)
+	if(ismob(user.pulling))
+		var/mob/M = user.pulling
```

Comment:
Need to figure out if we need this or not

Reply:
Do keep.